PRD.md

```diff
@@ -0,0 +1,30 @@
+# Final-project prd:
+
+## Purpose
+
+Discover locational patterns of major seismic events (earthquakes) around the world. Investigate the relationship between earthquake locations and things like tectonic plates or surrounding terrains.
+
+## Features
+
+This application is mainly going to be displayed as a website with a interative map that shows live time earthquake data. Functions may include:
+
+* Map marker for each earthquakes with
+    * Color of the marker showing the depth
+    * Size of the marker showing the magnitube
+* Clicking the marker will display details including:
+    * Location of event
+    * Time of occurance
+    * Depth of the earthquake
+    * Magnitude
```

Comment:
It might be interesting to display some historical information about "near-by" (say, within 100km :shrug:) earthquakes by [querying the USGS catalog](https://earthquake.usgs.gov/fdsnws/event/1/) as well -- maybe show on the map, or show some summary stats/graphs like a chart of magnitudes over time.